### ISSUE TITLE
Invalid options object. PostCSS

### ISSUE THREAD

**Issue by mirsella** [matzeso/quasar-app-extension-tailwindcss#15]
didn't manage to make it work with everything default with the latest quasar. Using yarn
I don't really know what config I can also upload, tell me

![image](https://user-images.githubusercontent.com/45905567/148546410-fd267497-43e4-425a-ac7d-1ce4113d3f97.png)

**Comment by lopugit** [comment]
I get this too 
```
28|subber-dev  | ValidationError: Invalid options object. PostCSS Loader has been initialized u  sing an options object that does not match the API schema.
28|subber-dev  |    - options has an unknown property 'plugins'. These properties are valid:
28|subber-dev  |      object { postcssOptions?, execute?, sourceMap?, implementation? }
```

**Comment by LiamSarsfield** [comment]
Seems to be the same issue happening in [this thread](https://github.com/laravel-mix/laravel-mix/issues/2471)
With [this fix ](https://github.com/laravel-mix/laravel-mix/issues/2471#issuecomment-701557916) I no longer get this error.

**Comment by matzeso** [comment]
Merged the PR and published in `2.0.6` hope this fixes eit. If not please reopen.
